lib/lte_link_control/lte_lc.c

```diff
@@ -1102,18 +1132,30 @@ int lte_lc_nw_reg_status_get(enum lte_lc_nw_reg_status *status)
 {
 	int err;
 	uint16_t status_tmp;
+	uint32_t cell_id;
```

Comment:
Because the AT+CEREG response does not always contain a cell ID, this should be initialized to 0. Otherwise it can be uninitialized when `is_cellid_valid()` is called.

Reply:
Good catch! I also updated the return value check after `nrf_modem_at_scanf()` to check for `< 1` instead of `!= 1`